src/aws_rds_iam_token.erl

```diff
@@ -0,0 +1,67 @@
+-module(aws_rds_iam_token).
+-export([create/4]).
+
+-define(SIGNING_ID, <<"rds-db">>).
+-define(EMPTY_PAYLOAD_HASH, <<"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855">>).
+
+%%====================================================================
+%% API
+%%====================================================================
+-spec create(aws_client:client(), binary(), non_neg_integer(), binary()) -> {ok, binary()}.
+create(Client, DBEndpoint, DBPort, DBUser) when is_map(Client) andalso
```

Comment:
Function name spaces are totally flat in Erlang, so this is not a good function name to export. I would strongly recommend you change this to `rds_token_create` or something like that.